client/src/modules/shopping-list/index.jsx

```diff
@@ -23,32 +35,80 @@ class ShoppingList extends Component {
     fetchItemsFromGivenList(id);
   };
 
+  showDialogBox = () => {
+    this.setState({ showDialogBox: true });
+  };
+
+  hideDialogBox = () => {
+    this.setState({ showDialogBox: false });
+  };
```

Comment:
Yes, I tried. It was working, but... it's not quite clear code because we invoke this methods in different places and for example it looks unclear and strange when we invoke toggleVisibility by cancelButton (this button have to do one thing: hide DialogBox). There is a little bit more code in my approach but I think it's more understandable... Of course I can change it but please consider my arguments